Update root transition operation page (#8494)
istio/istio.io: content/en/docs/ops/configuration/security/root-transition/index.md

test: n/a
---

{{< tip >}}

> You can skip this guide if your cluster started with Istio V1.3 or later versions,

Reviewer: ```suggestion
You can skip this guide if your cluster was started with Istio version 1.3 or later,
```

or if you do not use the Istio self-signed certificates.
{{< /tip >}}

> Before Istio V1.3, the Istio self-signed certificates had a 1 year default lifetime.
> If your cluster started with Istio V1.2 or earlier version,

Reviewer: ```suggestion
Before version 1.3, Istio self-signed certificates had a 1 year default lifetime.
If your cluster started with Istio version 1.2 or earlier,
```

and it is using Istio self-signed certificates,
you need to be mindful about the expiration date of the root certificate.

> The expiration of a root certificate may lead to an unexpected cluster-wide outage.
> 
> To evaluate the lifetime remaining for your root certificate, please refer to the first step in the
> [procedure below](#root-transition-procedure).

Reviewer: ```suggestion
The expiration of a root certificate may lead to an unexpected cluster-wide outage.
```

Author: Removed line 24.

> The steps below show you how to examine the remaining lifetime for your root certificate,

Reviewer: ```suggestion
The following steps show you how to examine the remaining lifetime for your root certificate,
```

    Execute the remainder of the steps prior to root certificate expiration to avoid system outages.

1. Execute a root certificate transition:

>     From Istio V1.5, Envoy can automatically load the new root certificate when it refreshes its

Reviewer: ```suggestion
    Starting with Istio version 1.5, Envoy can automatically load the new root certificate when it refreshes its
```

        }
    {{< /text >}}

>     Please inspect the `valid_from` value of `ca_cert`.

Reviewer: ```suggestion
    Inspect the `valid_from` value of `ca_cert`.
```
